exporters/otlp/internal/opentelemetry-proto-gen/resource/v1/resource.pb.go

```diff
@@ -1,7 +1,7 @@
 // Code generated by protoc-gen-gogo. DO NOT EDIT.
 // source: opentelemetry/proto/resource/v1/resource.proto
 
-package v1
+package v1 // import "go.opentelemetry.io/otel/exporters/otlp/internal/opentelemetry-proto-gen/resource/v1"
```

Comment:
Should this be modified?

Reply:
I think it might be a fools errand to try and updated these given it is generated. It will be reverted on the next upgrade. Not having this be included on this internal package seems fine to me.